File: bannerlayout/src/main/java/com/wkz/bannerlayout/listener/OnBannerClickListener.java
package com.wkz.bannerlayout.listener;

import android.support.annotation.NonNull;
import android.view.View;

/**
 * Banner Click event, object for the return of data
 */
public interface OnBannerClickListener {

    void onBannerClick(@NonNull View view, int position, BannerModelCallBack model);

}

File: bannerlayout/src/main/java/com/fphoenixcorneae/bannerlayout/listener/OnBannerClickListener.kt
package com.fphoenixcorneae.bannerlayout.listener

import android.view.View

/**
 * Banner Click event, object for the return of data
 */
interface OnBannerClickListener {
    fun onBannerClick(view: View, position: Int, model: BannerModelCallBack<*>?)
}